docs/en/protocols/README.md

```diff
@@ -1,51 +1,46 @@
-# Protocols
-There are two different types of protocols.
-
-- [**Probe Protocol**](#probe-protocols). It includes descriptions and definitions on how agents send collected metrics data and traces, as well as the format of each entity.
-
-- [**Query Protocol**](#query-protocol). The backend enables the query function in SkyWalking's own UI and other UIs. These queries are based on GraphQL.
-
+# Probe Protocol
+It includes descriptions and definitions on how agents send collected metrics, logs, traces and events, as well as the format of each entity.
 
 ## Probe Protocols
 They also related to the probe group. For more information, see [Concepts and Designs](../concepts-and-designs/overview.md).
 These groups are **language-based native agent protocol**, **service mesh protocol** and **3rd-party instrument protocol**.
 
-### Language-based native agent protocol
-There are two types of protocols that help language agents work in distributed environments.
-1. **Cross Process Propagation Headers Protocol** and **Cross Process Correlation Headers Protocol** come in in-wire data format. Agent/SDK usually uses HTTP/MQ/HTTP2 headers
-to carry the data with the RPC request. The remote agent will receive this in the request handler, and bind the context with this specific request.
-1. **Trace Data Protocol** is in out-of-wire data format. Agent/SDK uses this to send traces and metrics to SkyWalking or other compatible backends. 
+### Tracing
+There are two types of protocols that help language agents work in distributed tracing.
+
+- **Cross Process Propagation Headers Protocol** and **Cross Process Correlation Headers Protocol** come in in-wire data format. Agent/SDK usually uses HTTP/MQ/HTTP2 headers
+to carry the data with the RPC request. The remote agent will receive this in the request handler, and bind the context with this specific request. 
 
 [Cross Process Propagation Headers Protocol v3](Skywalking-Cross-Process-Propagation-Headers-Protocol-v3.md) has been the new protocol for in-wire context propagation since the version 8.0.0 release.
 
 [Cross Process Correlation Headers Protocol v1](Skywalking-Cross-Process-Correlation-Headers-Protocol-v1.md) is a new in-wire context propagation protocol which is additional and optional. 
-Please read SkyWalking language agents documentation to see whether it is supported. 
-This protocol defines the data format of transporting custom data with `Cross Process Propagation Headers Protocol`.
-It has been supported by the SkyWalking javaagent since 8.0.0, 
+Please read SkyWalking language agents documentation to see whether it is supported.
+
+- **Trace Data Protocol** is an out-of-wire data format. Agent/SDK uses this to send traces to SSkyWalking OAP server.
```

Comment:
- **Trace Data Protocol** is an out-of-wire data format. Agent/SDK uses this to send traces to SkyWalking OAP server.

Reply:
Fixed.